### PR TITLE
Fix whatdidiorder for OOO

### PR DESCRIPTION
**Pull request by alinden**

### CODE REVIEW

File: raybot/commands/whatdidiorder.rb [hunk 1]
```diff
@@ -7,31 +7,37 @@ class WhatDidIOrder < SlackRubyBot::Commands::Base
       match (/^.*(whatdidiorder).*$/) do |client, data, match|
         samaya = Samaya.new
 
-        my_name = client.store.users[data.user]["profile"]["last_name"]
-        break unless my_name
+        name = client.store.users[data.user]["profile"]["last_name"]
+        break unless name
```
Comment (rhollinger) [inline raybot/commands/whatdidiorder.rb]: probably worth some sort of error message here so ppl know why it's not working?

Comment (alinden) [reply raybot/commands/whatdidiorder.rb]: Done. This also apparently didn't work before cause "" is truthy in ruby.

File: raybot/commands/whatdidiorder.rb [hunk 2]
```diff
@@ -26,7 +29,7 @@ class WhatDidIOrder < SlackRubyBot::Commands::Base
         elsif order_status == "passed"
           client.say(channel: data.channel, text: "You passed on lunch today.")
           return
-        else
+        elsif order_status == "error" || order_status == "uninvited"
```
Comment (rhollinger) [inline raybot/commands/whatdidiorder.rb]: Are there other options?

**Comment by rhollinger** [comment]
Do you think this should reply in the channel it was asked? or in a DM to asker?

File: raybot/commands/whatdidiorder.rb (hunk 2 above)
Comment (alinden) [reply raybot/commands/whatdidiorder.rb]: "decided"

**Comment by alinden** [comment]
Okay, it'll now only respond to `what did i order` if asked in an im channel

File: raybot/commands/whatdidiorder.rb [hunk 3]
```diff
@@ -4,34 +4,52 @@
 module RayBot
   module Commands
     class WhatDidIOrder < SlackRubyBot::Commands::Base
-      match (/^.*(whatdidiorder).*$/) do |client, data, match|
+      match (/^.*([wW]hat\s+.*did\s+.*[iI]\s+.*order).*$/) do |client, data, match|
```
Comment (rhollinger) [inline raybot/commands/whatdidiorder.rb]: What is the point of this now? Now `whatdidiorder` doesnt work....

Comment (alinden) [reply raybot/commands/whatdidiorder.rb]: fixed

**Pull request merged**